packages/pixelboy/apps/journal/checkin/checkin.html

```diff
@@ -19,9 +19,14 @@
           <input class="image-file" type="file"/>
       </label>
       <label class="entry">
-        <p>Alternatively paste a link to an image or public post (tag it with #PixelArtAcademy for kudos points).</p>
+        <p>Alternatively paste a link to an image or public post (tag it with #PixelArtAcademy for kudos points). Currently, you can only get images from Twitter.</p>
         <input class="external-url" type="text"/>
       </label>
+      <h4>Image Preview</h4>
+      <div>
+        <img src="http://placehold.it/350x150?text=your+art" class="image-preview" style="max-width: 100%;">
```

Comment:
We typically do not use `style` unless it changes dynamically. So this should be in the stylus file.